### PR TITLE
Upgrade a bunch of JavaScript packages

### PR DESCRIPTION
**Pull request by yvanzo**
Mostly small game.

### CODE REVIEW

**Comment by yvanzo** [comment]
@brainzbot, retest this please.

**Comment by mwiencek** [comment]
I think there are some breaking changes in shelljs. The rest looks good though.

**Comment by yvanzo** [comment]
@brainzbot, retest this please.

File: webpack.client.config.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ function createJsPo(srcPo, lang) {
    * possible terminal paths.
    */
   const scriptsDir = shellQuote.quote([dirs.SCRIPTS]);
-  const nestedDirs = shell.exec(`find ${scriptsDir} -type d`, {silent: true}).output.split('\n');
+  const nestedDirs = _(shell.exec(`find ${scriptsDir} -type d`, {silent: true}).stdout).split('\n');
```
Comment (mwiencek) [inline webpack.client.config.js:111]: I don't think we need to create another lodash chain here. Did you run into a case where that's needed?
```suggestion
  const nestedDirs = shell.exec(`find ${scriptsDir} -type d`, {silent: true}).stdout.split('\n');
```

Comment (mwiencek) [reply webpack.client.config.js:111]: Maybe when stdout is a Buffer. I'd probably use `String()` over `_()` for clarity of intent though.

Comment (yvanzo) [reply webpack.client.config.js]: I thought that was needed but that was just a syntax error, thanks.

**Pull request merged**